samples/previews/aws-lambda/sqs/SQSLambda_1/AwsLambda.SQSTrigger/Function.cs

```diff
@@ -0,0 +1,56 @@
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+
+using Amazon.Lambda.Core;
+using Amazon.Lambda.SQSEvents;
+
+using NServiceBus;
+
+// Assembly attribute to enable the Lambda function's JSON input to be converted into a .NET class.
+[assembly: LambdaSerializer(typeof(Amazon.Lambda.Serialization.SystemTextJson.DefaultLambdaJsonSerializer))]
+
+public class Function
+{
+  public Function()
+  {
+  }
+
+  #region FunctionHandler
+  public async Task FunctionHandler(SQSEvent evnt, ILambdaContext context)
+  {
+    using (var cancellationTokenSource = new CancellationTokenSource(context.RemainingTime.Subtract(DefaultRemainingTimeGracePeriod)))
+    {
+      await serverlessEndpoint.Process(evnt, context, cancellationTokenSource.Token);
+    }
+  }
+  #endregion
+
+  static readonly TimeSpan DefaultRemainingTimeGracePeriod = TimeSpan.FromSeconds(10);
+
+  #region EndpointSetup
+
+  private static readonly IAwsLambdaSQSEndpoint serverlessEndpoint = new AwsLambdaSQSEndpoint(context =>
+  {
+    var endpointConfiguration = new AwsLambdaSQSEndpointConfiguration("AwsLambdaSQSTrigger");
+    endpointConfiguration.UseSerialization<NewtonsoftJsonSerializer>();
+
+    var transport = endpointConfiguration.Transport;
+    var routing = endpointConfiguration.RoutingSettings;
+
+    routing.RouteToEndpoint(typeof(BackToSenderMessage), "AwsLambda.Sender");
+
+    var advanced = endpointConfiguration.AdvancedConfiguration;
+    advanced.SendFailedMessagesTo("ErrorAwsLambdaSQSTrigger");
```

Comment:
Since we push this up to AWS to test, It could be the default will already exist for something else.  It makes sense to me that we use something that is specific to the sample.